packages/perseus/src/widgets/interactive-graphs/interactive-graph-regression.stories.tsx

```diff
@@ -433,15 +434,36 @@ export const LockedFiguresWithThickWeight: Story = {
     },
 };
 
+export const TooltipsWithFloatingPointIssues: Story = {
+    args: {
+        question: interactiveGraphQuestionBuilder()
+            .withPoints(5)
+            .withTooltips(true)
+            .withXRange(0, 5)
+            .withYRange(0, 3)
+            .withTickStep(1, 0.5)
+            .withGridStep(1, 0.1)
+            .withSnapStep(0.5, 0.1)
+            .build(),
+    },
+};
+
 function MafsQuestionRenderer(props: {question: PerseusRenderer}) {
     const {question} = props;
     return (
-        <Renderer
-            strings={mockStrings}
-            content={question.content}
-            widgets={question.widgets}
-            images={question.images}
-            apiOptions={ApiOptions.defaults}
-        />
+        <UserInputManager widgets={question.widgets} problemNum={0}>
```

Comment:
This is a fix for a regression caused by the introduction of the new UserInputManager, which was causing our Interactive Graph Regression tests to be missing key properties (such as graph types).